tuxemon/core/tools.py

```diff
@@ -165,6 +166,31 @@ def load_sprite(filename, **rect_kwargs):
     return sprite
 
 
+def load_animated_sprite(filenames, delay, **rect_kwargs):
+    """ Load a set of images and return an animated pygame sprite
+
+    Image name will be transformed and converted
+    Rect attribute will be set
+
+    Any keyword arguments will be passed to the get_rect method
+    of the image for positioning the rect.
+
+    :param filename: Filename to load
+    :rtype: core.sprite.Sprite
```

Comment:
docstring for delay.
dunno about kwargs

Reply:
Fixed for delay. The original get_sprite doesn't have a comment for the rect either and I followed its model.

Reply:
```python
:param int delay: ...
```

Reply:
Please reread the comment above.  it is missing "int"